src/dump-handlers.js

```diff
@@ -37,14 +36,17 @@ exports.mysql = function * (url, filename) {
   yield exec('tar', ['-czf', filename, 'dump'])
   return filename
 }
-
+// Back up a PostgreSQL database in a file call dump.sql, create an archive and compress it
 exports.postgres = function * (url, filename) {
+  console.log('>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>')
```

Comment:
Remove the logs : it would display the password in commandline = very bad practice